concent_api/concent_api/urls.py

```diff
@@ -13,14 +13,28 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
+from django.conf      import settings
 from django.conf.urls import url, include
 from django.contrib   import admin
 
 import core.urls
 import gatekeeper.urls
 
-urlpatterns = [
-    url(r'^',        include(gatekeeper.urls, namespace = 'gatekeeper')),
-    url(r'^api/v1/', include(core.urls, namespace = 'core')),
-    url(r'^admin/',  admin.site.urls),
-]
+urlpatterns = []
+
+assert hasattr(settings, 'CONCENT_FEATURES'), "CONCENT_FEATURES variable must be defined in django settings."
```

Comment:
This needs to be changed to a normal `if` that raises `ImproperlyConfigured` exception. `urls.py` is loaded before our Django checks run so we can't assume that missing `CONCENT_FEATURES` is bug. It can be a configuration error on user's part.